test/fixtures/deploy-full-suite.fixture.ts

```diff
@@ -252,6 +252,7 @@ export async function deploySuiteWithModularCompliancesFixture() {
 
   const complianceProxy = await ethers.deployContract('ModularComplianceProxy', [context.authorities.trexImplementationAuthority.address]);
   const compliance = await ethers.getContractAt('ModularCompliance', complianceProxy.address);
+  await compliance.bindToken(context.suite.token.address);
```

Comment:
Bind token to the compliance in the test itself (not in the generic fixture file)